caravel/views.py

```diff
@@ -1425,6 +1429,31 @@ class CssTemplateModelView(CaravelModelView, DeleteMixin):
     category_icon='')
 
 
+class SqlMacroModelView(CaravelModelView, DeleteMixin):
+
+    """View for list/add/edit SQL expression macros"""
+
+    datamodel = SQLAInterface(models.SqlExprMacro)
+    list_columns = ['namespace']
+    edit_columns = ['namespace', 'source_code']
+    add_columns = edit_columns
+    description_columns = {
+        'source_code': utils.markdown(
+            "a valid jinja2 template including a list of "
+            "[macros](http://jinja.pocoo.org/docs/dev/templates/#macros). "
+            "Example: `{% macro func(args) %} ... {% endmacro %}`", True)
+    }
```

Comment:
Similarly to what I mentioned before, recompiling in updates will only update jinja env in current worker, so it doesn't help in a multiple worker deployment.